charts/fluid/fluid/templates/controller/dataset_service.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if .Values.dataset.controller.statistics.enabled -}}
```

Comment:
Maybe the service is not necessary. I think we can connect to ip address of pod directly.